public/css/teststyle.css

```diff
@@ -0,0 +1,208 @@
+*{
```

Comment:
I think this looks pretty good; could you just separate this into a `merch.css` (merch-page specific stuff) and a `newstyle.css` (things that the general site would need if we updated the style, such as navbar things)